site/docs/latest/admin-api/namespaces.md

```diff
@@ -144,7 +144,7 @@ test-tenant/ns2
 
 #### REST API
 
-{% endpoint GET /admin/namespaces/:tenant/:cluster %}
+{% endpoint GET /admin/v2/namespaces/:tenant/:cluster %}
```

Comment:
This handle was removed in v2 API, so we should remove from here

Reply:
So the entire "List namespaces within a cluster" section ought to be deleted?

Reply:
@grantwwu Yes, that operation does not exist anymore in v2, please remove

Reply:
done